test/EstateRegistry.js

```diff
@@ -508,7 +538,91 @@ contract('EstateRegistry', accounts => {
+      await estate.updateManyLandData(
+        estateId,
+        [0, 0],
+        [1, 2],
+        'newValue',
+        sentByUser
+      )
+      const landsData = await Promise.all([
+        land.landData(0, 1, sentByUser),
+        land.landData(0, 2, sentByUser)
+      ])
+
+      landsData.forEach(data => data.should.be.equal('newValue'))
+    })
+    it('should allow operator of an Estate to update LANDs data', async function() {
+      await land.assignMultipleParcels([0, 0], [1, 2], user, sentByCreator)
+      const estateId = await createEstate([0, 0], [1, 2], user, sentByUser)
+      await estate.setApprovalForAll(anotherUser, true, sentByUser)
+      await estate.updateManyLandData(
+        estateId,
+        [0, 0],
+        [1, 2],
+        'newValue',
+        sentByAnotherUser
+      )
+      const landsData = await Promise.all([
+        land.landData(0, 1, sentByUser),
+        land.landData(0, 2, sentByUser)
+      ])
+
+      landsData.forEach(data => data.should.be.equal('newValue'))
+    })
+    it('should allow update operator of an Estate to update LANDs data', async function() {
+      await land.assignMultipleParcels([0, 0], [1, 2], user, sentByCreator)
+      const estateId = await createEstate([0, 0], [1, 2], user, sentByUser)
+      await estate.setUpdateOperator(1, anotherUser, sentByUser)
+      await estate.updateManyLandData(
+        estateId,
+        [0, 0],
+        [1, 2],
+        'newValue',
+        sentByAnotherUser
+      )
+      const landsData = await Promise.all([
+        land.landData(0, 1, sentByUser),
+        land.landData(0, 2, sentByUser)
+      ])
+
+      landsData.forEach(data => data.should.be.equal('newValue'))
+    })
+    it('should not allow neither operator nor owner nor updateOperator of an Estate to update LANDs data', async function() {
+      await land.assignMultipleParcels([0, 0], [1, 2], user, sentByCreator)
+      const estateId = await createEstate([0, 0], [1, 2], user, sentByUser)
+      await assertRevert(
+        estate.updateManyLandData(
+          estateId,
+          [0, 0],
+          [1, 2],
+          'newValue',
+          sentByAnotherUser
+        )
+      )
+    })
+    it('should not allow old owner to update LANDs data after creating an Estate', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await createEstate([0], [1], user, sentByUser)
+      await assertRevert(
+        land.updateManyLandData([0], [1], 'newValue', sentByUser)
+      )
+    })
+    it('should not allow old operator to update LANDs data after creating an Estate', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await land.setApprovalForAll(anotherUser, true, sentByUser)
+      await createEstate([0], [1], user, sentByAnotherUser)
+      await assertRevert(
+        land.updateManyLandData([0], [1], 'newValue', sentByAnotherUser)
```

Comment:
I don't understand this two tests (sorry 😅 )

`should not allow old owner to update LANDs data after creating an Estate `
`should not allow old operator to update LANDs data after creating an Estate`

Why aren't you supllying `estateId` to `updateManyLandData` ?

Reply:
As we are using an internal function _doTransferFrom oncreateEstate which cleans the updateOperator for the token being transferred I thought it was useful to have that test there. Once we implement the updateLandData on the EstateRegistry we will have a good set of tests for this subject